src/pages/api/modules/usenet/nzbget/types.ts

```diff
@@ -0,0 +1,27 @@
+export interface NzbgetHistoryItem {
+  NZBID: number;
+  Kind: 'NZB' | 'URL' | 'DUP';
+  NZBFilename: string;
+  Name: string;
+  URL: string;
+  HistoryTime: number;
+  DestDir: string;
+  FinalDir: string;
+  Category: string;
+  FileSizeLo: number;
+  FileSizeHi: number;
+  FileSizeMB: number;
```

Comment:
Another question: Are the fields of the SABnzbd and NZBget similar? If they have overlapping fields, we should extract them to a base type.

Reply:
From what I can tell, they are not really the same. The nzbget api returns more raw data and the caller has to do the logic to derive meaning (such as percent complete). The sabnzbd api seems to return more of this data already calculated.